src/main/telemetry/smartport.c

```diff
@@ -458,6 +458,10 @@ static void smartPortSendMspResponse(uint8_t *data) {
 
 void processSmartPortTelemetry(smartPortPayload_t *payload, volatile bool *clearToSend, const uint32_t *requestTimeout)
 {
+#if !defined(USE_MSP_OVER_TELEMETRY)
+    UNUSED(payload);
+#endif
+
```

Comment:
This is redundant and already implemented a few lines down - on the `#else` of the `#if defined(USE_MSP_OVER_TELEMETRY)`

Reply:
@hydra: Are you going to fix this?

Reply:
yes, fixed.